pyproject.toml

```diff
@@ -7,3 +7,21 @@ write_to = "src/asdf_standard/_version.py"
 
 [tool.black]
 line-length = 120
+force-exclude = '''
+^/(
+  (
+      \.eggs
+    | \.git
+    | \.pytest_cache
+    | \.tox
+    | reference_files
+    | build
+    | dist
```

Comment:
Should we be excluding the resources directory here?

Reply:
Again, resources is just yaml files so it will be ignored anyways.